vcd/resource_vcd_nsxt_edgegateway.go

```diff
@@ -156,6 +156,14 @@ func resourceVcdNsxtEdgeGateway() *schema.Resource {
 				Optional:    true,
 				Description: "Dedicating the External Network will enable Route Advertisement for this Edge Gateway.",
 			},
+			"deployment_mode": {
```

Comment:
I wonder if this will make `vcd.ResourceSchema-vcd_nsxt_edgegateway.tf` upgrade test fail?

Reply:
Yes! Will add

Reply:
Added